src/Control/Lens/Prism.hs

```diff
@@ -132,13 +132,13 @@ outside k = withPrism k $ \bt seta f ft ->
 without :: APrism s t a b
         -> APrism u v c d
         -> Prism (Either s u) (Either t v) (Either a c) (Either b d)
-without k =
-  withPrism k         $ \bt seta k' ->
-  withPrism k'        $ \dv uevc    ->
-  prism (bimap bt dv) $ \su ->
-  case su of
-    Left s  -> bimap Left Left (seta s)
-    Right u -> bimap Right Right (uevc u)
+without k uvcd =
+  (withPrism k         $ \bt seta k' ->
```

Comment:
Can anyone think of a cleaner way to accomplish this?

Reply:
Manually inlining `withPrism` definition...

Reply:
Or just

```
without :: APrism s t a b
        -> APrism u v c d
        -> Prism (Either s u) (Either t v) (Either a c) (Either b d)
without k k' =
   withPrism k         $ \bt seta ->
   withPrism k'        $ \dv uevc    ->
   prism (bimap bt dv) $ \su ->
   case su of
     Left s  -> bimap Left Left (seta s)
     Right u -> bimap Right Right (uevc u)
```

Reply:
https://github.com/ekmett/lens/pull/942#discussion_r498707113 is a fantastic idea. Suggestion applied.